source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -17,12 +17,13 @@ Format
 
 Each YAML file has the following keys:
 
-- description: Some text.
+- description: A textual description of the test.
 - uri: A connection string.
 - phases: An array of "phase" objects.
+  A phase of the test optionally sends inputs to the client,
+  then tests the client's resulting TopologyDescription.
 
-A "phase" of the test sends inputs to the client, then tests the client's
-resulting TopologyDescription. Each phase object has two keys:
+Each phase object has two keys:
```

Comment:
Per my comment below, I think this should read "has the following keys" and then "(optional)" should be added after the "responses" key in a manner consistent with how we document optional fields in other specs (see: https://github.com/mongodb/specifications/tree/master/source/crud/tests).